chapter_26_page_pattern.asciidoc

```diff
@@ -16,9 +16,9 @@ and let me know how you enjoy it!
 ((("functional tests (FTs)", "with multiple users", secondary-sortas="multiple users", id="FTmultiple25")))
 ((("functional tests (FTs)", "structuring test code", id="FTstructure25")))
 Are jokes about how "everything has to be social now" slightly old hat?
-Yes Harry, they were old hat 10 years ago when you started writing this book,
+Yes Harry, they were old hat 10 years ago when you started writing the first edition of fthis book,
 and they're positively prehistoric now.
-Irregardless, let's say lists are often better shared.
+Regardless, let's say lists are often better shared.
```

Comment:
aw that's my little joke way of spelling regardless.  can i keep it?  maybe if i make it clearer that it's a joke,rather than a typo?